src/audiovideocontroller/DefaultAudioVideoController.ts

```diff
@@ -294,7 +294,7 @@ export default class DefaultAudioVideoController
   }
 
   private uninstallPreStartObserver(): void {
-    this.meetingSessionContext.signalingClient?.removeObserver(this.preStartObserver);
+    this.meetingSessionContext.signalingClient.removeObserver(this.preStartObserver);
```

Comment:
Undoes https://github.com/aws/amazon-chime-sdk-js/pull/2299/files which is no longer needed due to removal below. Not possible to unit test (but the previous test added still exists)